src/DataFields.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace mikehaertl\pdftk;
+
+use ArrayObject;
+
+/**
+ * Class DataFields
+ *
+ * @author Ray Holland <raymondaholland+php-pdftk@gmail.com>
+ */
+class DataFields extends ArrayObject
+{
+    private $_string;
+
+    private $_array;
+
+    /**
+     * DataFields constructor.
+     *
+     * @param string $input
+     * @param int $flags
+     * @param string $iterator_class
+     */
+    public function __construct($input = null, $flags = 0, $iterator_class = "ArrayIterator")
+    {
+        $this->_string = $input ?: '';
```

Comment:
@bravadomizzou The shortand `?:` is only available since PHP 5.3. It's a pretty old version, I know. But changing this to the "long" version `$input ? $input : ''` would make it compatible with older versions, too - with very little extra effort. Could you also change this here and in the other locations please?

And sorry that I didn't see it in my first review.